generator/main.h

```diff
@@ -55,20 +55,16 @@ struct Preprocess
         QStringList includes;
         includes << QString(".");
 
-#if defined(Q_OS_WIN32)
-        const char *path_splitter = ";";
-#else
-        const char *path_splitter = ":";
-#endif
+        QChar pathSplitter = QDir::listSeparator();
 
-        // Environment INCLUDE
-        QString includePath = getenv("INCLUDE");
+        // Environment PYTHONQT_INCLUDE
+        QString includePath = getenv("PYTHONQT_INCLUDE");
         if (!includePath.isEmpty())
-            includes += includePath.split(path_splitter);
+            includes += includePath.split(pathSplitter);
 
         // Includes from the command line
         if (!commandLineIncludes.isEmpty())
-            includes += commandLineIncludes.split(path_splitter);
+            includes += commandLineIncludes.split(pathSplitter);
```

Comment:
Perhaps add parameter Qt::SkipEmptyParts?

Reply:
Since we just had the case: What about adding a check if the given include directories actually exist? (This should include QTDIR!)